src/app/styles/components/_related-products.less

```diff
@@ -0,0 +1,29 @@
+//
+// related products
+//
+.c-related-products {
+    padding-left: 45px;
+    padding-right: 45px;
```

Comment:
this padding should be declared with a variable. try `@grid-gutter-width * 1.5` (i believe @grid-gutter-width is 30px)